src/utils/date.ts

```diff
@@ -0,0 +1,21 @@
+import { parseDate } from "@h6s/calendar";
+
+export const isDateInFuture = (date: Date): boolean => {
+  const today = new Date();
```

Comment:
Why not simply converting to epoch and compare two integers?

Reply:
I think don't even need to convert it, we could just compare `Date` objects directly. I can definitely simplify this.